src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_COST = 40;
+  const SHORT_TERM = 3;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM_DISCOUNT = 20;
+  const totalRentalCost = days * RENT_COST;
+
+  if (days >= LONG_TERM) {
+    return totalRentalCost - LONG_TERM_DISCOUNT;
+  }
+
+  if (days >= SHORT_TERM) {
+    return totalRentalCost - SHORT_TERM_DISCOUNT;
+  }
+
+  return totalRentalCost;
 }
 
 module.exports = calculateRentalCost;
```

Comment:
The function is properly exported using `module.exports`, allowing it to be used in other parts of the application or in tests.